frontend/src/components/Menu/Menu.js

```diff
@@ -1,28 +1,40 @@
 import "./Menu.scss";
-import { Link } from "react-router-dom";
-import { Button } from "@material-ui/core";
 import React from "react";
+import { Button } from "@material-ui/core";
+import { useHistory } from "react-router";
 
 const Menu = () => {
+  const history = useHistory();
+
+  const redirect = path => {
```

Comment:
same as above, this function is redundant unless you are planning to do something more inside it's body.